README.md

```diff
@@ -18,41 +18,29 @@ With bp, cFS can participate in a Delay/Disruption Tolerant Network (DTN) as a D
 
 #### Prerequisites
 
-1. The build requires the __cmake__ build system and a compiler toolchain (by default __gcc__).
+1. The build requires Ubuntu 22.04.4 LTS, the __cmake__ build system and a compiler toolchain (by default __gcc__).
```

Comment:
See comment on bplib PR.

Reply:
From the comment on bplib PR: Recommend something along the lines of "The bp application has been tested on Ubuntu 22.04.4 LTS, although it is expected to work on other POSIX-compliant operating systems."
I used "bp" instead of "BP" because they are introduced separately in the Overview to mean "the cFS Bundle Protocol Application (bp)" and "Bundle Protocol (BP)".